tests/contrib.bats

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "bud with a path to a Dockerfile (-f) containing a non-directory entry" {
+  run_buildah bud -t buildah ../contrib/buildahimage/stable
+  run_buildah from buildah
+  run_buildah run $output rpm -qV shadow-utils
```

Comment:
This is probably a really stupid question, but might it not be better to run this `rpm -qV` in the magic github-workflow script that creates the remote images?

Reply:
An then blow up there?  My goal is to catch changes that can end up with  a bad image, when users checkin fixes to the contrib Dockerfiles.